src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -111,10 +112,12 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Remark updatedRemark = editPersonDescriptor.getRemark().orElse(personToEdit.getRemark());
         DateOfBirth updatedDateOfBirth = editPersonDescriptor.getDateOfBirth().orElse(personToEdit.getDateOfBirth());
         Income updatedIncome = editPersonDescriptor.getIncome().orElse(personToEdit.getIncome());
+        // edit command does not allow editing appointments
```

Comment:
Noted. I think this info is important and should be mentioned at the top of the class.